src/main/java/uk/gov/ons/census/fwmt/jobservice/http/comet/CometRestClient.java

```diff
@@ -65,7 +65,7 @@ public CometRestClient(
     this.basePath = cometUrl + "{}";
     this.createPath = cometUrl + "{}";
     this.closePath = cometUrl + "{}/close";
-    this.deletePath = cometUrl + "{}";
+    this.deletePath = cometUrl + "{}/delete";
```

Comment:
check the spec again, its a pausePath but with a DELETE verb

Reply:
changed to {}/pause